db/migrate/20181207191632_devise_create_users.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
```

Comment:
We can avoid this by adding the following statement to the .rubocop.yml file:
```
Style/FrozenStringLiteralComment:
  Enabled: false
```

Reply:
Also rubocop should not pass on migration file.